### ISSUE TITLE
autosave only works when project root has .flint folder

### ISSUE THREAD

**Issue by knightli** [flintjs/flint-atom2#23]
- dir1
  - .flint
  - xx.js
  - main.js

when atom project home is dir1, autosave works.

- dir2
  - dir1
    - .flint
    - xx.js
    - main.js

when atom project home is dir2, autosave not work (not save on type, save on click or other trigger atom's native save action)

**Comment by ncammarata** [comment]
yeah, we've noticed this but we've seen most people have their Flint app as their project directory. Can you explain a bit more about how you structure things?
